.circleci/config.yml

```diff
@@ -71,6 +71,16 @@ jobs:
       - run:
           name: Smoke tests
           command: sbt -Dfatal-warnings=true -Dscalac-optimize=true clean test docker:stage exit
+      - run:
+          name: Save test results
+          command: |
+            mkdir -p ~/junit/
+            find . -type f -regex ".*/target/test-reports/TEST-.*xml" -exec cp {} ~/junit/ \;
```

Comment:
Do you need the first `.*`?
Or is the `target` directory on the working directory?

Reply:
Yes, target is in the working directory. I followed the example from circlecidocs for junit - https://circleci.com/docs/2.0/collect-test-data/#gradle-junit-test-results